test/functional/services/browser.ts

```diff
@@ -37,7 +37,7 @@ export async function BrowserProvider({ getService }: FtrProviderContext) {
 
   const isW3CEnabled = (driver as any).executor_.w3c === true;
 
-  if (!isW3CEnabled) {
+  if (browserType === Browsers.Chrome) {
```

Comment:
From Chromedriver 75 release notes: `Added new endpoints for retrieving Chrome log` 
So it works in both W3C compliant and "old" modes, I tested it.